src/taipy/core/data/abstract_file.py

```diff
@@ -26,3 +28,19 @@ def _build_path(self, storage_type):
         if not dir_path.exists():
             dir_path.mkdir(parents=True, exist_ok=True)
         return dir_path / f"{self.id}.{self.__EXTENSION_MAP.get(storage_type)}"
+
+    @staticmethod
+    def _check_and_update_preserve_file(old_path: Optional[str] = None, new_path: Optional[str] = None):
```

Comment:
Since it is not the responsibility of the abstract_file_data_node to preserve the preserve_file, I would move this code to a dedicated class/module.

Reply:
I thought about moving it to the common module as it's outside of the main logic of Core. I'm only keeping it in the abstract file because only file data nodes need to preserve the path. What do you think?

Reply:
I would move it to a new package named `_backup`, in a `_backup.py` module.
And I would rename `_check_and_update_preserve_file` to `_backup_files`.